db/migrate/20220104140658_use_encrypted_data_for_account_credentials.rb

```diff
@@ -10,7 +10,9 @@ def up
 
     LegacyAccountCredentials.reset_column_information
 
-    LegacyAccountCredentials.find_each do |a|
+    # ose secrets are migrated during decrypt (encrypted_data)
+    # so only load credentials here
```

Comment:
I don't think this has anything to do with what you are doing ? Maybe just incorrectly rebased or had something in local changes that you accidentally added?

Reply:
I agree that this has nothing to do with what we are doing, but if I revert the change the migration spec for this file gives an error.
What should we do with this?

Reply:
Seems like this isn't relevant anymore? As I don't see the change anymore and the specs pass :grin: